sc-kpm/sc-agents-common/generated/ANumberComparison.generated.hpp

```diff
@@ -0,0 +1,69 @@
+#include <memory>
```

Comment:
Generated files shouldn't be commited. They will be generated during build process